libs/typings/src/ui/plugins.ts

```diff
@@ -1,19 +1,25 @@
-import { GetUserInfo, IUserStore } from './store';
+import { IUserStore } from './store';
 
-export interface IPluginsMap {
-  [namespace: string]: any;
-}
-
-export interface PluginConf {
+/**
+ * Interface defining plugin configuration object
+ */
+export interface IPluginConf {
   [namespace: string]: {
-    [key: string]: unknown;
+    [key: string]: any;
   };
 }
 
+/**
+ * Type defining param of a function which fetches profile info
+ * @internal
+ **/
+type GetUserInfo = {
```

Comment:
sounds more like the function itself. could we use ProfileInfoParams?

Reply:
Addressed.